includes/User/MySimsPageBuilder.php

```diff
@@ -54,14 +71,14 @@ public function build_html() {
                         <ul class="my-esims-page-content-list" id="my-esims-page-content-list">
                             <li>
                                 <input type="checkbox" class="my-esims-page-content-list-checkbox" name="usage-content" id="my-esims-page-list-usage" />
-                                <label for="my-esims-page-list-usage" class="my-esims-page-list-title">Usage</label>
+                                <label for="my-esims-page-list-usage" class="my-esims-page-list-title">'.$this->t('my.esims.usage').'</label>
                                 <div class="my-esims-page-list-desc">
                                 ' . $this->get_usage_data( $iccid ) . '
                                 </div>
                             </li>
                             <li>
                                 <input type="checkbox" class="my-esims-page-content-list-checkbox" name="installation-content" id="my-esims-page-list-installation" />
-                                <label for="my-esims-page-list-installation" class="my-esims-page-list-title">Installation</label>
+                                <label for="my-esims-page-list-installation" class="my-esims-page-list-title">'.$this->t('my.esims.installation').'</label>
```

Comment:
please add some spacing between the string concatenation, this will trigger the WP checks unfortonately